### PR TITLE
[ZT] Update "Replace your VPN" learning path

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/_partials/warp/_add-local-domain-IP.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+In **DNS Servers**, enter the IP address of the DNS server that should resolve that domain name.
+
+   - All servers are tried and the fastest response is used. If this response is `no records found`, then that response will be used and the other servers will not be queried.
```
Comment (maxvp) [inline content/cloudflare-one/_partials/warp/_add-local-domain-IP.md]: Nit: This sentence feels passive voice heavy.

File: content/cloudflare-one/_partials/warp/_device-enrollment-mtls.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+To check for an mTLS certificate:
+
+1. [Add an mTLS certificate](/cloudflare-one/identity/devices/access-integrations/mutual-tls-authentication/#add-mtls-authentication-to-your-access-configuration) to your account. You can generate a sample certificate using the [Cloudflare PKI toolkit](/cloudflare-one/identity/devices/access-integrations/mutual-tls-authentication/#test-mtls-using-cloudflare-pki).
+
+2. In **Associated hostnames**, enter your Zero Trust {{<glossary-tooltip term_id="team domain">}}team domain{{</glossary-tooltip>}}: `<team-name>.cloudflareaccess.com`
+3. In your [device enrollment permissions](#set-device-enrollment-permissions), add a _Common Name_ or _Valid Certificate_ rule. For example, the following policy requires a client certificate with a specific common name:
+
+    | Action   | Rule type | Selector    | Value                 |
+    | -------- | --------- | ------------| --------------------- |
+    | Allow    | Require   | Common Name |  `<CERT-COMMON-NAME>` |
```
Comment (maxvp) [inline content/cloudflare-one/_partials/warp/_device-enrollment-mtls.md:17]:
```suggestion
    | Allow    | Require   | Common Name | `<CERT-COMMON-NAME>`  |
```

Comment (maxvp) [inline content/cloudflare-one/_partials/warp/_device-enrollment-mtls.md]:
```suggestion
    | Action   | Rule type | Selector    | Value                |
    | -------- | --------- | ------------| -------------------- |
    | Allow    | Require   | Common Name | `<CERT-COMMON-NAME>` |
```

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/device-profiles.md [hunk 3]
```diff
@@ -25,6 +28,38 @@ At this time, **Split Tunnels** and **Local Domain Fallback** can only be modifi
...
+  "name": "Cloudflare basic device profile",
+  "precedence": 101,
+  "service_mode_v2": {
+    "mode": "warp"},
+  "support_url": "https://it.company.com/help",
+  "switch_locked": true
+}'
+```
+
+{{</tab>}}
```
Comment (maxvp) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/device-profiles.md]: Should we add additional context for using this method? E.g. `Make the following API call.` before the code block.

File: content/cloudflare-one/connections/connect-networks/private-net/cloudflared/_index.md [hunk 4]
```diff
@@ -37,25 +37,41 @@ By default, all WARP devices enrolled in your Zero Trust organization can connec
...
+6. Configure your [App Launcher](/cloudflare-one/applications/app-launcher/) visibility and logo.
+7. Select **Next**. You will see two auto-generated Gateway Network policies: one that allows access to the destination IP and another that blocks access.
+8. Modify the policies to include additional identity-based conditions. For example:
 
-### Create Zero Trust policies
+   - **Policy 1**
+     | Selector       | Operator      | Value            | Logic | Action |
+     | -------------- | ------------- | ---------------- | ----- | ------ |
+     | Destination IP | in            | `10.128.0.7`     | And   | Allow  |
+     | User email     | Matches regex | `.*@example.com` |       |        |
```
Comment (maxvp) [inline content/cloudflare-one/connections/connect-networks/private-net/cloudflared/_index.md]:
```suggestion
     | User Email     | matches regex | `.*@example.com` |       |        |
```

File: content/learning-paths/replace-vpn/build-policies/create-policy.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
+
+| Traffic Selector | Operator | Value                  |
+| -------- | -------- | ---------------------- |
+| Domain   | in list  | `Company Wiki domains` |
+
+| Identity Selector | Operator | Value                  |
+| -------- | -------- | ---------------------- |
+| User email  | Matches regex  | `.*@example.com` |
```
Comment (maxvp) [inline content/learning-paths/replace-vpn/build-policies/create-policy.md]:
```suggestion
| User email  | matches regex  | `.*@example.com` |
```

File: content/learning-paths/replace-vpn/build-policies/create-policy.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
+
+| Traffic Selector | Operator | Value                  |
+| -------- | -------- | ---------------------- |
+| Destination IP   | in list  | `Company Wiki IPs` |
+
+| Identity Selector | Operator | Value                  |
+| -------- | -------- | ---------------------- |
+| User email  | Matches regex  | `.*@example.com` |
```
Comment (maxvp) [inline content/learning-paths/replace-vpn/build-policies/create-policy.md]:
```suggestion
| User Email  | matches regex  | `.*@example.com` |
```

**Review by maxvp — APPROVED** [review]

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/device-profiles.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/configure-warp/device-profiles.md]: Yes, good call.

**Pull request merged**